src/main/java/com/scribassu/tracto/domain/ExamPeriodEvent.java

```diff
@@ -0,0 +1,25 @@
+package com.scribassu.tracto.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class ExamPeriodEvent {
+    private enum TypeOfEvent {
```

Comment:
make enum public and move it to separate file and rename to EventType (or maybe ExamPeriodEventType)

Reply:
+